src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,110 @@
+package duke;
+
+import duke.exceptions.FileIoException;
+import duke.tasks.*;
+import duke.io.Loader;
+import duke.io.Saver;
+import duke.exceptions.EmptyDescriptionException;
+import duke.exceptions.UnknownCommandException;
+
+
+import java.util.ArrayList;
```

Comment:
I think `import java.*` should be before `import duke.*`, according to the checkStyle function.